### ISSUE TITLE
Training Merlin with full data set

### ISSUE THREAD

**Issue by kl2792** [CSTR-Edinburgh/merlin#169]
Hello,

I was wondering if there's any way to train Merlin on a full training data set (i.e. with `Test=0` in `global_settings.cfg`).

I have my own label files in `test_synthesis` that I want to use for testing, and it seems that Merlin doesn't output a model when I do set the test size to 0, and it nondeterministically doesn't output a model when I set the test size to 1. Only when I put 1/12 of the training data into the test set does Merlin consistently output a model.

Please let me know if anyone know a way to work around this.

Thanks so much!
